contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -251,10 +253,11 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
    */
   struct Upkeep {
     bool paused;
+    bool overridesEnabled;
```

Comment:
Note: this newly added field will not exist in `IAutomationRegistryMaster2_3.sol -> UpkeepInfoLegacy`. But this shouldnt be a backward compatible issue.